.github/workflows/e2e.yml

```diff
@@ -2,14 +2,22 @@ name: Run e2e tests
 
 on:
   workflow_dispatch:
-
+    inputs:
+      run-target:
+        description: Command line for running e2e tests
+        default: 'yarn e2e:ci'
+        required: true
+      platforms:
+        description: OS platforms to test on (list of strings in JSON format)"
+        default: '["ubuntu-20.04"]'
+        required: true
 jobs:
   run-e2e-tests:
     name: Run e2e tests
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        os: [ubuntu-20.04]
+        os: ${{ fromJSON(inputs.platforms) }}
```

Comment:
I learned this trick from https://github.com/actions/toolkit/issues/184#issuecomment-1666870406